### DISCUSSION TITLE
Casual hi!

### DISCUSSION THREAD

**Discussion by goyalyashpal** [noahgorstein/jqp#53] [General]
i was just yeasterday using `jq` & really thought that pipiing to jq again & again doesnt seem right. t'd have been awesome if there were a TUI for this. or a dedicated console sessions like in sql queries.

and found it today via this blog: 
https://slott56.github.io/2023-11-14-this_is_awful.html

'will try to use it if installable on windows

**Comment by noahgorstein** [comment]
thanks very much! glad you're finding it helpful 😸

**Comment by goyalyashpal** [comment]
> 'will try to use it if installable on windows

yeah, have gotten around setting up scoop package manager. and `jqp` is available there :)
'have installed it and off i go to use it :D

https://github.com/noahgorstein/jqp/blob/8e3b58efea8684837123f1c76a696f8e1ed49105/README.md?plain=1#L68

```console
$ scoop search jq
Results from local buckets...

Name Version Source Binaries
---- ------- ------ --------
jqp  0.5.0   extras
jq   1.7.1   main

$ scoop info jqp

Name        : jqp
Description : A TUI playground to experiment with jq
Version     : 0.5.0
Bucket      : extras
Website     : https://github.com/noahgorstein/jqp
License     : MIT
Updated at  : 2024-01-18 22:50:34
Updated by  : Aliaksandr Belik
Binaries    : jqp.exe


```
